README.md

```diff
@@ -29,14 +29,12 @@ samples:
 
     # Unseeded partitions go here
     partition-file: /path/to/Hs-LN-D-5RACE-IgG/partition.csv
-    cluster-annotation-file: /path/to/Hs-LN-D-5RACE-IgG/partition-cluster-annotations.csv
```

Comment:
@psathyrella We still need `cluster-annotation-file` if it's csv data right?

Reply:
no -- since the only code that will ever decide on the relative naming of the partition and cluster annotation files is in partis, I just moved the code that duplicates that in cft into a fcn in partis utils. This is especially true sincein the future, there'll only be the one yaml file.

Reply:
OK; I'm generally not keen on making assumptions related to naming conventions. But if this is how partis is working now, and its just for the old format, I can live with it.

Reply:
However, it might be nice if we could add some content to the README explaining that if you have data in the old partis format, the files can't be moved around or renamed in any way which might violate this assumption.